docs/src/components/APILink/index.tsx

```diff
@@ -25,8 +25,8 @@ export function APILink({ fn, children }: { fn: string; children?: React.ReactNo
 
   const docLink = useBaseUrl(`/${APIModules[module]}#${fn}`);
   return (
-    <Link to={docLink} target="_blank">
+    <a href={docLink} target="_blank">
```

Comment:
I'm not sure if using a tag is the right approach here. Won't we lose the ability to detect broken links when we use a tag? Instead, can't we redirect api doc links with trailing slash or consolidate the path policy (display api doc correctly with trailing slash)?

Reply:
I believe broken links check only works for Docusaurus internal links, not API reference links. I can add a redirect too, to catch any missing links, but I think it would be more consistent to use `<APILink>` component for all links to API reference.

Also I should probably add a lint rule to use the component for API reference links.

Reply:
I reverted back to using `<Link>` and intentionally broke a link to check, didn't get a broken links error during build. Seems like we're not losing any safety here.

Reply:
Got it. Let's go with `a` tag then. Thanks for checking